src/crawlee/parsel_crawler/_utils.py

```diff
@@ -0,0 +1,63 @@
+from __future__ import annotations
+
+import re
+
+from parsel import Selector
+
+from crawlee._utils.html_to_text import (
+    _ANY_CONSECUTIVE_WHITE_SPACES,
+    _EMPTY_OR_ENDS_WITH_ANY_WHITE_SPACE,
+    _EMPTY_OR_ENDS_WITH_NEW_LINE,
+    BLOCK_TAGS,
+    SKIP_TAGS,
+)
+
+
+def html_to_text(source: str | Selector) -> str:
+    """Converts markup string or `Selector` to newline-separated plain text without tags using Parsel."""
```

Comment:
I would use the same docstring title as for the BS variant. And maybe mention some specific details in the further description?